.schema/latest.json

```diff
@@ -293,6 +296,30 @@
             "null"
           ]
         },
+        "git_only": {
+          "title": "Git Only",
+          "description": "If true, use git tags to determine the latest version instead of the registry.\nIf unspecified at package level, inherits from workspace config.",
```

Comment:
"if unspecified.." that's true for all the configurations, so I wonder if it makes sense to specify it.

Maybe we can delete that sentence